src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.ts

```diff
@@ -386,4 +395,22 @@ export class InstructorHomePageComponent extends InstructorSessionModalPageCompo
   downloadSessionResultEventHandler(tabIndex: number, rowIndex: number): void {
     this.downloadSessionResult(this.courseTabModels[tabIndex].sessionsTableRowModels[rowIndex]);
   }
+
+  /**
+   * Retries loading all the courses
+   */
+  retryLoadingCourses(): void {
+    this.hasCoursesLoadingFailed = false;
+    this.hasCoursesLoaded = false;
+    this.loadCourses();
+  }
+
+  /**
+   * Retries loading a feedback session from a course
+   */
+  retryLoadingSession(index: number): void {
```

Comment:
Similar here and in other places also. What do you think?

Reply:
I think that would work for the places where the retry calls one load only. Then the retry component just calls that load method. But there are some retry that does multiple load method calls like in `student-course-details-page`. I think the ones with multiple load calls would still benefit from having a retry method for both readability and ensuring the variable is set to false at one point only.

Reply:
@xpdavid So what do you think about the suggestion above(i.e., single load be shifted but we keep retry method for multiple load calls)?

Reply:
Yup. That would make more sense.